lib/core/client.js

```diff
@@ -628,10 +629,11 @@ class NightwatchClient extends EventEmitter {
 
   setCurrentTest() {
     this.setApiProperty('currentTest', () => this.reporter.currentTest);
+    this.setApiProperty('currentSection', () => this.reporter.currentSection);
```

Comment:
is it necessary to expose this API? Didn't we add section for our internal use (separate hooks data from test data)

Reply:
Not needed, fixed.